src/cache/lru_cache_manager/lru_cache_manager.h

```diff
@@ -23,18 +23,18 @@ typedef struct LRUCacheManager
  * @param  capacity: Cache cacpcity
  * @retval Pointer to the cache manager
  */
-LRUCacheManager *LRUCacheManager_New(size_t capacity);
+LRUCacheManager *LRUCacheManager_New(size_t capacity, cacheValueFreeFunc freeCB);
```

Comment:
Missing comment for the cacheValueFreeFunc argument.

Reply:
file removed